### PR TITLE
Fixed bug: Do not fail if scheme is missing from IPMI url

### PR DESCRIPTION
**Pull request by neoaggelos**
By default, `urlparse('hostname.domain.ext').netloc` returns an empty value, because the protocol scheme is missing. This means that even though `https://my.ipmi.address`  works, `my.ipmi.address` returns an empty string.

### CODE REVIEW

File: maas2netbox/validators.py [hunk 1]
```diff
@@ -131,8 +130,8 @@ def check_nodes(self):
         for node in self.netbox_nodes:
             try:
                 if node['custom_fields']['IPMI']:
-                    declared_ipmi_address = urlparse(
-                        node['custom_fields']['IPMI']).netloc
+                    declared_ipmi_address = utils.get_hostname(
```
Comment (skatsaounis) [inline maas2netbox/validators.py]: Please move the method inside this class as a static method rather than creating a new file

**Review by skatsaounis — CHANGES_REQUESTED** [review]

File: maas2netbox/validators.py (hunk 1 above)
Comment (neoaggelos) [reply maas2netbox/validators.py]: Added as a static method on the base class Validator.

File: maas2netbox/validators.py [hunk 2]
```diff
@@ -14,13 +14,14 @@
...
 
 import logging
-from urllib.parse import urlparse
 
 from maas.client.enum import NodeStatus
 
 from maas2netbox import config
 from maas2netbox.utils import ipmi, maas, netbox
 
+from urllib.request import urlparse
```
Comment (skatsaounis) [inline maas2netbox/validators.py]: please revert the change of import. It used to be present at line 17

**Review by skatsaounis — CHANGES_REQUESTED** [review]

File: maas2netbox/validators.py (hunk 2 above)
Comment (neoaggelos) [reply maas2netbox/validators.py]: Should be OK now

File: maas2netbox/validators.py [hunk 3]
```diff
@@ -14,7 +14,7 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 import logging
-from urllib.parse import urlparse
+from urllib.request import urlparse
```
Comment (skatsaounis) [inline maas2netbox/validators.py]: This change will only work because urllib.request imports urlparse: https://github.com/python/cpython/blob/3.8/Lib/urllib/request.py#L103

Please revert it to import the urlparse function from urllib.parse which is the place where it is defined

**Review by skatsaounis — CHANGES_REQUESTED** [review]

**Review by skatsaounis — APPROVED** [review]
LGTM
